src/main/java/com/sandvoxel/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -207,15 +205,39 @@ public long getPower() {
 
 
     @Override
-    public void update() {
-
-        //container.givePower(1000,false);
-
+    public void update()
+    {
 
+//      checking for the machine type
         if (machineTier == null)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3ACommentedOutCodeLine)